src/abstract/hash-to-curve.ts

```diff
@@ -127,7 +127,7 @@ export function expand_message_xof(
  */
 export function hash_to_field(msg: Uint8Array, count: number, options: Opts): bigint[][] {
   validateObject(options, {
-    DST: 'string',
+    DST: 'stringOrUint8Array',
```

Comment:
I suppose you could make it take an array of validator keys?? and if .some(...) of them pass ??